predtimechart-config.yml

```diff
@@ -66,4 +65,4 @@ task_id_text:
     "69": "Northern Mariana Islands"
     "72": "Puerto Rico"
     "74": "U.S. Minor Outlying Islands"
-    "78": "Virgin Islands"
+    "78": "Virgin Islands"
```

Comment:
noting what looks like maybe a deletion of and endline? probably doesn't matter.